apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/MutateRowTest.cs

```diff
@@ -0,0 +1,359 @@
+﻿// Copyright 2017, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Bigtable.V2.IntegrationTests
+{
+    [Collection(nameof(BigtableFixture))]
+    public class MutateRowTest
+    {
+        private readonly BigtableFixture _fixture;
+
+        public MutateRowTest(BigtableFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public async Task AddNewRow()
+        {
+            var tableName = _fixture.DefaultTableName;
+            var client = _fixture.DefaultTableClient;
+            BigtableByteString rowKey = Guid.NewGuid().ToString();
+            client.MutateRow(
+                tableName,
+                rowKey,
+                Mutations.SetCell(
+                    BigtableFixture.ColumnFamily1,
+                    "column_name",
+                    "test12345",
+                    new BigtableVersion(1000)));
+
+            await VerifySingleRowAsync(
+                client,
+                tableName,
+                rowKey,
+                BigtableFixture.ColumnFamily1,
+                "column_name",
+                "test12345",
+                new BigtableVersion(1000));
+
+            // Cleanup
```

Comment:
Do we rely on this? I generally do all the clean-up when the fixture is disposed. If there's a good reason to clean up here for other tests, maybe add a comment explaining why?

Reply:
Not needed. You're right. removed.